.circleci/config.yml

```diff
@@ -404,6 +416,106 @@ references:
     - Demisto SDK Nightly - Run against Cortex XSOAR:
         context: sdk_nightly_env
 
+  bucket_upload_jobs: &bucket_upload_jobs
```

Comment:
Also need to verify that in case of failure in each of these jobs (unittests, validation, server 6.0 and marketplace) a slack message will be sent to the channel to notify on the failure.